### PR TITLE
Nested lists

### PR DESCRIPTION
**Pull request by joefitter**
* added slate-edit-list fork
* configured plugin

### CODE REVIEW

File: client/components/editor/index.js [hunk 1]
```diff
@@ -67,16 +67,12 @@ class TextEditor extends Component {
   };
 
   onBlur = (self, editor, next) => {
-    next();
+    next()
```
Comment (wheelsandcogs) [inline client/components/editor/index.js]: ;

Comment (wheelsandcogs) [inline client/components/editor/index.js]:
```suggestion
    next();
```

Comment (joefitter) [reply client/components/editor/index.js]: there are about a billion of these in projects, as our linting doesn't care, we should make it care, but let's do them all together?

File: assets/scss/_editor.scss [hunk 2]
```diff
@@ -185,3 +188,16 @@
...
+  position: relative;
+
+  &:after {
+    position: absolute;
+    content: '';
+    border: 2px solid #d9edff;
+    left: -25px;
+    height: 100%;
+    top: 0;
+  }
```
Comment (lennym) [inline assets/scss/_editor.scss]: What's occurring here?

Comment (joefitter) [reply assets/scss/_editor.scss]: its something I took from the example in the plugin - we may or may not want it, but it seems kind of useful when editing a list/nested list - it highlights the currently scoped list item on which the change will be performed

Comment (joefitter) [reply assets/scss/_editor.scss]: see here https://gitbookio.github.io/slate-edit-list/

Comment (joefitter) [reply assets/scss/_editor.scss]: happy to remove it if you think it's not needed

Comment (lennym) [reply assets/scss/_editor.scss]: No, I was just curious what it was for. Looks nice to me.

**Pull request merged**